internal/store/postgres/asset_repository.go

```diff
@@ -24,6 +29,27 @@ type AssetRepository struct {
 	defaultUserProvider string
 }
 
+type DeleteAssetSQLExpr struct {
+	queryexpr.SQLExpr
+}
+
+func (d *DeleteAssetSQLExpr) Validate() error {
+	identifiers, err := queryexpr.GetIdentifiers(d.String())
+	if err != nil {
+		return err
+	}
+
+	mustExist := generichelper.Contains(identifiers, "refreshed_at") &&
+		generichelper.Contains(identifiers, "type") &&
+		generichelper.Contains(identifiers, "service")
+	if !mustExist {
+		return fmt.Errorf("must exist these identifiers: refreshed_at, type, and service. "+
+			"Current identifiers: %v", identifiers)
+	}
+
+	return nil
+}
```

Comment:
this is redundant between es and sql, also let's validate if the identifiers is not supported, we need to list which property of assets is support which is not

Reply:
Done mas